store/kv/kv_store.go

```diff
@@ -79,26 +78,35 @@ func (tx *KeyValueTx) Read(ctx context.Context, table []byte, key Key) (Iterator
 	if err != nil {
 		return nil, err
 	}
-	return &KeyValueIterator{
-		baseIterator: iter,
-	}, nil
+	if config.DefaultConfig.Metrics.Fdb.Enabled {
```

Comment:
do we need it here? The TxImplWithMetrics has Read and ReadRange method so we can wrap up the iterator inside that?

Reply:
I don't think so, I will move it.